src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -451,14 +469,25 @@ def update(
                 "nqn": Param(str, "NVMeoF subsystem NQN"),
                 "nsid": Param(str, "NVMeoF Namespace ID"),
                 "gw_group": Param(str, "NVMeoF gateway group", True, None),
+                "force": Param(str, "Force remove the RBD image")
```

Comment:
```suggestion
                "force": Param(bool, "Force remove the RBD image")
```

Reply:
actually this is `str` as intended since we get that as a string in query_param and later convert it to boolean. Updated the param type as well.

Reply:
Okay, yeah in paramt type it was bool, so confused. Thanks